### ISSUE TITLE
NMRium in nmrXiv does not read NMRium files

### ISSUE THREAD

**Issue by tilfischer** [NFDI4Chem/nmrxiv#649]
**Is your enhancement request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Comment by NRayya** [comment]
Hi @tilfischer 
Could you please share the project with me on nmrXiv? It would be easier than duplicating the work

**Comment by tilfischer** [comment]
As ever before, the data is available in RADAR: https://dx.doi.org/10.22000/786

**Comment by tilfischer** [comment]
While NMRium does not read nmrium files added in Step 1 (File Upload) of a submission, NMRium itself in Step 2 is happy to read this files via Import -> Import file from file system.

That is how that may look like:

![nmrXiv_nmrrium_files](https://user-images.githubusercontent.com/85482437/213442523-dcc97432-5afe-4dc1-b0e7-013ad1ae3f51.png)

On the left you see data obtained with one sample. Data is available in Bruker format and in JCAMP-DX format. On the right in NMRium you see the data in NMRium format, which is not added as spectra listed on the left.

**Comment by NishaSharma14** [comment]
Duplicate #531

**Comment by tilfischer** [comment]
Issue still present in current version of nmrXiv. If I upload the nmrium file mentioned and linked above and want to proceed to step 2/3 I get the following error message ```Spectra files needs to be organised into folders. Please create a folder corresponding to each sample and add all your NMR spectroscopic experiment output files are added to the corresponding folders```. If I wrap the nmrium file with a folder I get ```Please upload spectral data to proceed.```. 

Possible this issue is of minor importance. The nmrium format is possibly of minor importance. The [description of the format](https://docs.nmrium.org/for-developers/nmrium_file_format) is very limited and there are vendor formats, JCAMP-DX formats, nmrML and formats by Mestrelab out there. That's possibly already more than enough.

**Comment by CS76** [comment]
Quick update:

TLDR;
NMRium format is constantly evolving and updated without retrospective compatibility. Therefore, we cannot enable this feature unless a stable release is provided by the core NMRium team.

At nmrXiv, we use sources and source selectors to link spectra instead of embedding them in the nmrium file due to file size issues affecting loading times. Parsing the NMRium file to extract metadata is possible. For example, in this project, we found zones defined for 2D spectra and correlation data.

**Comment by tilfischer** [comment]
This is totally a'right from my perspective. We do not need another exchange format but one which lab users have in their hand and which they may use to also provide data on peaks, integrals, multiplets/ranges and chemical structures + assignments.

The [documentation of the NMRium format](https://docs.nmrium.org/for-developers/nmrium_file_format/) is very limited anyhow. It seems to me that it is is not intended to be an exchange format.
